packages/terra-demographics-banner/src/_utils.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
Lets discuss in our meeting later today, I lean towards `_renderUtils.jsx` but understand the logic for calling it `_utils.jsx`